src/openid.js

```diff
@@ -14,16 +31,11 @@ const getUserInfo = accessToken =>
       // and http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
       const claims = {
         sub: `${userDetails.user.id}`, // OpenID requires a string
-        name: userDetails.user.real_name,
+        name: userDetails.user.name,
         preferred_username: userDetails.user.name,
-        email: userDetails.user.profile.email,
-        phone: userDetails.user.profile.phone,
-        picture: userDetails.user.profile.image_512,
-        updated_at: userDetails.user.updated,
-        team: userDetails.user.team,
-        website: userDetails.user.team,
-        given_name: userDetails.user.first_name,
-        family_name: userDetails.user.last_name
```

Comment:
some of these are not present in identity response… it's a tradeoff, but in my case at least, I rather ask for just identity and have less but easier data than asking too much permissions for this extra data